### PR TITLE
Added safety checks to clearTimeout. Closes #13.

### LINKED ISSUE #13: clearTimeout arguments
Allow falsy values passed into `clearTimeout` so it is consistent with the native implementation.

### PR DESCRIPTION
**Pull request by arb**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Timeout {
 
 const _setTimeout = (...args) => { return new Timeout(...args); };
 
-const _clearTimeout = (timer) => { return timer.close(); };
+const _clearTimeout = (timer) => { timer && timer.close && timer.close(); };
```
Comment (cjihrig) [inline src/index.js]: Could this be: `(timer instanceof Timeout) && timer.close();`

Comment (arb) [inline src/index.js]: I'm always leery of using `instanceOf`... do you have a strong case for it other than removing one `&&`?

Comment (cjihrig) [inline src/index.js]: I'm fine with keeping it this way. If you really want to be safe, you should change `timer.close` to `typeof timer.close === 'function'`. :-)

I guess you could go even further and verify that timer is a non-null object in the first place. Probably all overkill though.

**Comment by cjihrig** [comment]
LGTM.

**Pull request merged**